lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php

```diff
@@ -644,3 +644,7 @@
 $wgGroupPermissions['*']['fandom-admin'] = false;
 $wgGroupPermissions['staff']['fandom-admin'] = true;
 $wgGroupPermissions['fandom-editor']['fandom-admin'] = true;
+
+// request to be forgotten
+$wgGroupPermissions['*']['requesttobeforgotten'] = false;
+$wgGroupPermissions['staff']['requesttobeforgotten'] = true;
```

Comment:
I'd create a separate user group for this and add it to only those people who need it

Reply:
I don't think that's really necessary here. This special page is meant mostly for testing purposes and will be there until a generic solution for GDPR is created that will cover MediaWiki and other services.